commit 5e71cd7517cfa1906d2a51791a6eba327807d3ab
Author: Geoffrey M Gunter <geoffrey.m.gunter@gmail.com>
Date:   Wed Nov 3 23:12:27 2021 +0000

    Fix ENVIRaster test dataset for GDAL v3.3.3
    
    This update provides a more explicit WKT string for the ENVI raster
    dataset used in unit tests. This is necessary for GDAL v3.3.3 to be able
    to recognize the EPSG code. Previous versions of GDAL were able to
    correctly infer the EPSG code in my testing.

diff --git a/tests/data/io/gdal/ENVIRaster-dem.hdr b/tests/data/io/gdal/ENVIRaster-dem.hdr
index 1234fcf8..6ab00850 100644
--- a/tests/data/io/gdal/ENVIRaster-dem.hdr
+++ b/tests/data/io/gdal/ENVIRaster-dem.hdr
@@ -10,6 +10,6 @@ data type = 4
 interleave = bsq
 byte order = 0
 map info = {Geographic Lat/Lon, 1, 1, -156.000138888886, 20.0001388888836, 0.000277777777777815, 0.000277777777777815,WGS-84}
-coordinate system string = {GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]]}
+coordinate system string = {GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AXIS["Latitude",NORTH],AXIS["Longitude",EAST],AUTHORITY["EPSG","4326"]]}
 band names = {
 Band 1}
